src/actions/player/index.js

```diff
@@ -45,6 +45,13 @@ function emptyPlaylist() {
   };
 }
 
+function setIsInShuffleMode(isInShuffleMode) {
+  return {
+    type: actionTypes.CHANGE_SHUFFLE_MODE,
+    isInShuffleMode
```

Comment:
When you follow (1) you would have to remove the payload here and only trigger the action to flip the boolean which is already available in the reducer.